internal/core/graphics/image.rs

```diff
@@ -813,6 +813,19 @@ impl Image {
         })
     }
 
+    /// Returns the [WGPU](http://wgpu.rs) 24.x texture that this image wraps; returns None if the image does not
+    /// hold such a previously wrapped texture.
+    ///
+    /// *Note*: This function is behind a feature flag and may be removed or changed in future minor releases,
+    ///         as new major WGPU releases become available.
```

Comment:
i'm thinking it would be nice if all the wgpu function would link to the wpu-24 module docs for more info

Reply:
Good idea. From this note.